MonoGame.Framework.Content.Pipeline/MonoGame.Content.Builder.targets

```diff
@@ -84,10 +84,13 @@
           Command="$(MonoGameContentBuilderCmd) /@:&quot;%(ContentReferences.FullPath)&quot; $(Header) /outputDir:&quot;%(ContentReferences.ContentOutputDir)&quot; /intermediateDir:&quot;%(ContentReferences.ContentIntermediateOutputDir)&quot; /quiet"
           WorkingDirectory="%(ContentReferences.RootDir)%(ContentReferences.Directory)" />
 
-   
+
+    <CreateItem Include="%(ContentReferences.ContentOutputDir)\**\*.*" Condition="'%(ContentReferences.Link)' != ''" AdditionalMetadata="ContentOutputDir=$([System.String]::Copy('%(ContentReferences.Link)').Replace('%(Filename)%(Extension)', ''))">
```

Comment:
What I am doing here, is for `ContentReferences` that are `linked` files, then it will get it's `Link` metadata property to  work out what the path of it is within the project structure. I have to get rid of the filename itself as I only care about the folder it lives in. 

For non linked `ContentReferences` - the below CreateItem just uses the RelativeDir = which should be the relative dire

If the `ContentReference` is not a linked file, then I use the RelativeDir = which is the location of it within the project.